### PR TITLE
Added Interactive Quiz in Explore Page #52

### PR DESCRIPTION
**Pull request by Prashant-2024**
Issue No. #52 

- Interactive Quiz section added in Explore page.
- 10 random questions 
- questions are saved in constant.dart

### CODE REVIEW

**Comment by K123Ritesh** [comment]
@Prashant-2024 Resolve the merge conflicts

File: android/app/build.gradle [hunk 1]
```diff
@@ -54,30 +54,33 @@ android {
         applicationId "vdrs.sappu.lafk.learn"
         // You can update the following values to match your application needs.
         // For more information, see: https://docs.flutter.dev/deployment/android#reviewing-the-gradle-build-configuration.
-        minSdkVersion 21    
+        minSdkVersion 21
         targetSdkVersion flutter.targetSdkVersion
         versionCode flutterVersionCode.toInteger()
         versionName flutterVersionName
+        multiDexEnabled true
```
Comment (K123Ritesh) [inline android/app/build.gradle:61]: does Multidex Enabling Must @Prashant-2024

Comment (Prashant-2024) [reply android/app/build.gradle]: i recently upgraded my flutter, since then i had to add this in build

**Comment by Prashant-2024** [comment]
@K123Ritesh i don't know but suddenly i am failing a check for Greetings, 
This wasn't in my earlier PR, but it's causing trouble now.

Can you help me here?

**Pull request merged**